packages/@coorpacademy-progression-engine/src/config/index.js

```diff
@@ -3,10 +3,12 @@ import {find} from 'lodash/fp';
 import type {Config, Engine, Progression} from '../types';
 import microlearning from './microlearning';
 import learner from './learner';
+import externalConfig from './external';
```

Comment:
pk pas juste external ?

Reply:
`'external' is already declared in the upper scope  no-shadow`